static/js/components/subject_page/block.tsx

```diff
@@ -119,8 +120,13 @@ export function Block(props: BlockPropType): JSX.Element {
           props.columns.map((column, idx) => {
             const id = getId(props.id, COLUMN_ID_PREFIX, idx);
             const columnTileClassName = getColumnTileClassName(column);
+            let className = "";
+            if (idx >= minIdxToHide) {
+              className = HIDE_TILE_CLASS;
+            }
             return (
               <Column
+                className={className}
```

Comment:
nit: make a new `shouldHideTiles: boolean` prop instead so that it's clear what's happening?